app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  before_action :authenticate_user!
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password) }
+    devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :email, :password, :current_password) }
```

Comment:
Nice job so far @Ellyboi ! 👏 But some changes are required!
I tried to register on the registration page you created and it turned out there are no user controllers. 😂  Hence, the page is not working.🔧
However, nice css design! 👏
<img width="1435" alt="Screenshot 2023-08-02 at 9 03 41 AM" src="https://github.com/julzedz/recipe-app/assets/73921498/fbe3c52e-5abc-45a8-81fe-eff959383371">